.github/workflows/velox_be.yml

```diff
@@ -40,89 +40,92 @@ on:
 env:
   HTTP_PROXY_HOST: 10.240.252.16
   HTTP_PROXY_PORT: 911
-  DOCKER_REGISTRY_HOST: 10.0.2.4
+  DOCKER_REGISTRY_HOST: 10.0.0.25
   DOCKER_REGISTRY_PORT: 5000
-
+  PATH_TO_GLUTEN_TE: ./tools/gluten-te
+  PREVIOUS_DOCKER_REGISTRY_HOST: 10.0.2.4
+  
 concurrency:
   group: ${{ github.repository }}-${{ github.head_ref || github.sha }}-${{ github.workflow }}
   cancel-in-progress: true
 
 jobs:
-
   ubuntu2004-test-spark32:
     runs-on: velox-self-hosted
+    env:
+      OS_IMAGE: ubuntu
+      OS_VERSION: 20.04
     steps:
       - uses: actions/checkout@v4
```

Comment:
`- uses: actions/checkout@v4`

I think this step can be removed since we are now checking out code inside the docker via `checkout.sh`.